lib/trento/domain/value_objects/saptune_service_status.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Trento.Domain.SaptuneServiceStatus do
+  @required_fields [:enabled, :active, :name]
+
+  use Trento.Type
+
+  deftype do
+    field :name, :string
+    field :enabled, :boolean
```

Comment:
I thinks `enabled/active` should be of string type, and we should forward exactly what it comes from the raw output.

Reply:
refactored to the latest updates from the team